### ISSUE TITLE
Accessibility: Color contrast is poor when selecting editor text in high contrast mode

### ISSUE THREAD

**Issue by dsaw** [processing/p5.js-web-editor#3413]
### What is your operating system?

Windows

### Actual Behavior

When switching to high contrast mode in p5.editor.js, selecting the text using the cursor changes the selection background to a light color, but the text color is not changed in that selection. 

<img width="380" alt="Image" src="https://github.com/user-attachments/assets/88ad3bc3-7232-4d6d-9efb-4d50837e7458" />

### Expected Behavior

The color contrast should be as per recommended standard (https://webaim.org/resources/contrastchecker/) and the text should be clearly visible even in high contrast mode. The comment text is barely visible/readable for example.

### Steps:
1.Change p5.editor.js to high contrast mode in preference modal 2.Select a text in editor 3. View that the text is barely visible inside the selection as the selection rectange is light color.
### Snippet:
```js

// Paste your code here :)

```

**Comment by Harshit-7373** [comment]
@dsaw This issue has been resolved in this pull request :-  [https://github.com/processing/p5.js-web-editor/pull/3396](url)

The pull request has also been merged.
